src/statistical/unigram.jl

```diff
@@ -0,0 +1,184 @@
+"""
+struct Sentencepiecemodel
+  vocab::Array{String,1}
+  logprob::Array{Float64,1}
+end
+structure, To hold vocabulary,log probability and index
+    
+"""
+struct Sentencepiecemodel
+  vocab::Array{String,1}
+  logprob::Array{Float64,1}
+end
+function load(ty::Type{T}, name::String) where T<:PretrainedTokenizer
+        filepath = @datadep_str name
+        filepath = "$filepath/$name"
+        load(filepath)  
+end
+    
+
+function load(path)
+    vocab = readlines(path)
+    vocabnew = split.(vocab , "\t")
+    vo = []
+    for i in 1:30000
```

Comment:
Why 30000?

Reply:
If it is the vocab size?
Better to replace with the length, so that this same code can support different vocab sizes from various other sentencepiece trained tokenizers.

Reply:
sure